tests/try_scraper/test_invalid_revision.py

```diff
@@ -5,24 +5,18 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from mock import patch
```

Comment:
Would you mind replacing the usage of the plain mock package with https://pypi.org/project/pytest-mock/? This would also apply to the other files.

Reply:
I also wanted to use it. But was not sure whether to use it as mock was used in other files too. Your comment helps on that matter.